src/elements/featured-videos.html

```diff
@@ -26,6 +26,11 @@
 
       .title {
         font-size: 24px;
+        min-width: 400px;
```

Comment:
With `min-width` you will get horizontal scroll
![image](https://user-images.githubusercontent.com/2954281/27854058-f83b0744-616d-11e7-8f7f-9e2afa4e6597.png)